srcpkgs/nostr-rs-relay/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'nostr-rs-relay'
+pkgname=nostr-rs-relay
+version=0.8.9
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config protobuf"
+makedepends="openssl-devel"
+short_desc="Nostr relay"
+maintainer="Petr Cizmar <xxb0322@xpc.ovh>"
+license="MIT"
+homepage="https://git.sr.ht/~gheartsfield/nostr-rs-relay"
+changelog="https://git.sr.ht/~gheartsfield/nostr-rs-relay/refs"
+distfiles="https://git.sr.ht/~gheartsfield/nostr-rs-relay/archive/${version}.tar.gz"
+checksum=a497340760fc99745fdb42c7ea04013d25bc57e50ca4749887129361866a98fa
+system_accounts="_nostr"
+system_groups="_nostr"
```

Comment:
should probably use `_nostr_rs_relay` for user and group names just so that there won't be any future conflicts.

Reply:
Yep, that makes sense....